blockchain-agent/src/main/java/com/sphereon/alfresco/blockchain/agent/rest/model/ErrorResponse.java

```diff
@@ -0,0 +1,64 @@
+package com.sphereon.alfresco.blockchain.agent.rest.model;
+
+import com.google.common.base.MoreObjects;
+import com.google.gson.annotations.SerializedName;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+@ApiModel(
+        description = "The error response"
+)
+public class ErrorResponse {
+    @SerializedName("errors")
+    private List<Error> errors;
+
+    public ErrorResponse() {
+        this.errors = new ArrayList<>();
+    }
+
+    public ErrorResponse errors(List<Error> errors) {
+        this.errors = errors;
+        return this;
+    }
+
+    public ErrorResponse addErrorsItem(Error errorsItem) {
+        this.errors.add(errorsItem);
+        return this;
+    }
+
+    @ApiModelProperty("")
+    public List<Error> getErrors() {
+        return this.errors;
+    }
+
+    public void setErrors(List<Error> errors) {
+        this.errors = errors;
```

Comment:
Copy the list and thus protecting the argument?

Reply:
Done